src/mem/cache/tags/compressed_tags.cc

```diff
@@ -104,12 +116,18 @@ CompressedTags::tagsInit()
 CacheBlk*
 CompressedTags::findVictim(Addr addr, const bool is_secure,
                            const std::size_t compressed_size,
-                           std::vector<CacheBlk*>& evict_blks)
+                           std::vector<CacheBlk*>& evict_blks,
+                           const uint64_t partition_id=0)
 {
     // Get all possible locations of this superblock
-    const std::vector<ReplaceableEntry*> superblock_entries =
+    std::vector<ReplaceableEntry*> superblock_entries =
         indexingPolicy->getPossibleEntries(addr);
 
+    // Filter entries based on PartitionID
+    for (auto partitioning_policy : partitioningPolicies)
+        partitioning_policy->filterByPartition(superblock_entries,
+            partition_id);
```

Comment:
```suggestion
    for (auto partitioning_policy : partitioningPolicies) {
        partitioning_policy->filterByPartition(superblock_entries,
            partition_id);
    }
```

It scares me to see multiline if statements without curly braces. I've had to waste too many days tracking down bugs from code like this.

Reply:
Noted, this has been fixed in the latest version